crates/harnx-vercel-grep-server/README.md

```
@@ -0,0 +1,96 @@
+          "language": "python",
+          "code_snippet": "..."
+        }
+      ]
+    }
+  ]
+}
+```
+
+**Note:** grep.app is currently behind a Vercel bot-challenge that returns HTTP 429 to some automated/datacenter clients. Depending on your network environment, a live query may return:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)